examples/install-clusterwide.yml

```diff
@@ -58,9 +67,9 @@ roleRef:
   kind: ClusterRole
   name: excoredns
 subjects:
-- kind: ServiceAccount
-  name: excoredns
-  namespace: kube-system
+  - kind: ServiceAccount
+    name: excoredns
+    namespace: kube-system
 ---
```

Comment:
is this meant to be indented?

Reply:
All of these files were limited using vscode yaml linter